extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -63,6 +63,7 @@ import {insertAnalyticsElement} from '../../../src/extension-analytics';
 import {
   getAdSenseAmpAutoAdsExpBranch,
 } from '../../../ads/google/adsense-amp-auto-ads';
+import {validateAdSenseData} from '../../../ads/google/utils';
```

Comment:
this introduces a dependency to `3p/` directory, which we want to avoid.

there is a [TODO](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js#L626) in doubleclick impl related to this as well. It might make sense to move the `validateData` function out of `3p/` for sharing.

@keithwrightbos any thoughts where we should move this?

Reply:
Hey @lannka, @keithwrightbos, any updates on this?